app/src/db/bulk_ops.py

```diff
@@ -0,0 +1,150 @@
+"""Bulk database operations for performance.
+
+Provides a bulk_upsert function
+"""
+from typing import Any, Optional, Sequence
+
+import psycopg
```

Comment:
Should we explicitly include psycopg in Poetry dependencies file rather than rely on it being included by SQLAlchemy? 🤔

Reply:
I believe it already is -- https://github.com/navapbc/template-application-flask/pull/201/files#diff-f79644217c104a26d0814d8faf2afc24c0239350806e80ff239b93271888e465R22